File: test/language/expressions/cpp/ModuloOperatorTest.cpp
#include "expressions/modulo_operator/ModuloFunction.h"
#include "gtest/gtest.h"

namespace expressions
{
namespace modulo_operator
{

TEST(ModuloOperatorTest, isValueDivBy4)
{
    const ModuloFunction moduloFunction;
    ASSERT_TRUE(moduloFunction.funcIsValueDivBy4());
}

} // namespace modulo_operator
} // namespace expressions

File: test/language/expressions/java/expressions/ModuloOperatorTest.java
package expressions;

import static org.junit.jupiter.api.Assertions.*;

import org.junit.jupiter.api.Test;

import expressions.modulo_operator.ModuloFunction;

public class ModuloOperatorTest
{
    @Test
    public void isValueDivBy4()
    {
        final ModuloFunction moduloFunction = new ModuloFunction();
        assertEquals(true, moduloFunction.funcIsValueDivBy4());
    }
}
